hack/.tmp.tidy-check/go.mod

```diff
@@ -0,0 +1,79 @@
+module antrea.io/antrea
```

Comment:
this file?

Reply:
has deleted, caused by rebase and 'make test-tidy'